docs/developer-docs/latest/plugins/sentry.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Sentry - Strapi Developer Docs
+description: Send email from your server or externals providers.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/plugins/email.html
+---
+
+# Sentry
+
+This plugin enables you to track errors in your Strapi application using [Sentry](https://sentry.io/welcome/).
+
+By using the Sentry plugin you can:
+
+* Initialize a Sentry instance upon startup of a Strapi application
+* Send Strapi application errors as events to Sentry
+* Include additional metadata in Sentry events to assist in debugging
+* Expose a [global Sentry service](#global-sentry-service)
+
+## Installation
+
+Install the Sentry plugin by adding the dependency to your Strapi application as follows:
+
+<code-group>
+
+<code-block title="NPM">
+```sh
+npm install @strapi/plugin-sentry
+```
+</code-block>
+
+<code-block title="YARN">
+```sh
+yarn add @strapi/plugin-sentry
+```
+</code-block>
+
+</code-group>
+
+Then you must [configure](#configuration) the plugin to enable your Strapi application to send events to the Sentry instance.
```

Comment:
Not sure about that paragraph. Perhaps it would be better to transform that into some kind of introduction for the "Configuration" section no? I think it's better to only have information related to the installation in the "Installation" section (and same thing for "Configuration" etc.).
Or if you want to explain the logical order of the documentation/how to make the Sentry plugin work from start to finish, you can add that in the introduction at the beginning of the documentation.
What do you think?